spot-client/src/spot-tv/ui/components/meeting-frame/ApiHealthCheck.test.js

```diff
@@ -0,0 +1,97 @@
+import ApiHealthCheck from './ApiHealthCheck';
+
+describe('ApiHealthCheck', () => {
+    let apiHealthCheck;
+    let mockJitsiApi;
+    let onErrorCallback;
+
+    beforeEach(() => {
+        jest.useFakeTimers();
+
+        mockJitsiApi = {
+            isVideoMuted: jest.fn().mockReturnValue(Promise.resolve())
+        };
+
+        onErrorCallback = jest.fn();
+
+        apiHealthCheck = new ApiHealthCheck(mockJitsiApi, onErrorCallback);
+    });
+
+    afterEach(() => {
+        apiHealthCheck.stop();
+    });
+
+    describe('start', () => {
+        it('it no-ops if already started', () => {
```

Comment:
this is not a no-op but throws

Reply:
Right. Bug from the rename destroy to stop change.